cmd/notation/inspect.go

```diff
@@ -247,6 +245,11 @@ func getCertificates(outputFormat string, envContent *signature.EnvelopeContent)
 func printOutput(outputFormat string, ref string, output inspectOutput) error {
 	if outputFormat == cmd.OutputJSON {
 		return ioutil.PrintObjectAsJSON(output)
+	} else {
+		if len(output.Signatures) == 0 {
```

Comment:
fixed

```shell
➜  notation git:(no-sig) ✗ ./notation inspect $IMAGE 
Warning: Always inspect the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
localhost:6000/net-monitor@sha256:52cc9cf0f2e0cadf49337acc7a45a07a2ce5a0ef37702efb0a851884bc32b7b1 has no associated signature

➜  notation git:(no-sig) ✗ ./notation inspect $IMAGE -o json
Warning: Always inspect the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
{
    "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
    "Signatures": []
}

```